pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /bin
```

Comment:
@the-lazy-learner what is this file for? If this is for your local setting no need to push this. Please git ignore and remove this.